Library/Homebrew/cask/lib/hbc/download_strategy.rb

```diff
@@ -10,6 +11,10 @@ module Hbc
   class AbstractDownloadStrategy
     attr_reader :cask, :name, :url, :uri_object, :version
 
+    APP_NAME = "Homebrew-Cask".b.freeze
+    TYPE = format("%04d", (LSQuarantine::Database::TYPE_NUMBERS[:web_download]).freeze)
+    STATUS = format("%04d",  (81).freeze)
```

Comment:
I got this constant empirically (downloading lots of apps via Firefox and Safari and seeing which value corresponded to an unopened package).